duckdb_queries/places.sql

```diff
@@ -26,7 +26,7 @@ COPY (
         AND bbox.maxx < -122.2477071
         AND bbox.miny > 47.5621587
         AND bbox.maxy < 47.7120663
-    ) TO 'places_seattle.geojsonseq'
+    ) TO 'places_seattle.geojsonl'
 WITH (FORMAT GDAL, DRIVER 'GeoJSONSeq');
```

Comment:
should be
WITH (FORMAT GDAL, DRIVER 'GeoJSONSeq', **SRS 'EPSG:4326');**

as explained here https://github.com/duckdb/duckdb_spatial/issues/141

Reply:
Thanks for pointing this out, I'll go ahead and add it to the examples